src/textual/widgets/_collapsible.py

```diff
@@ -1,87 +1,93 @@
 from __future__ import annotations
 
-from textual.widget import Widget
+from rich.console import RenderableType
+from rich.text import Text
 
 from .. import events
 from ..app import ComposeResult
-from ..containers import Container, Horizontal
+from ..binding import Binding
+from ..containers import Container
+from ..css.query import NoMatches
 from ..message import Message
 from ..reactive import reactive
 from ..widget import Widget
-from ..widgets import Label
 
-__all__ = ["Collapsible"]
+__all__ = ["Collapsible", "CollapsibleTitle"]
 
 
-class Collapsible(Widget):
-    """A collapsible container."""
-
-    collapsed = reactive(True)
+class CollapsibleTitle(Widget, can_focus=True):
+    """Title and symbol for the Collapsible."""
 
     DEFAULT_CSS = """
-    Collapsible {
-        width: 1fr;
+    CollapsibleTitle {
+        width: auto;
         height: auto;
+        padding: 0 1 0 1;
+    }
+
+    CollapsibleTitle:hover {
+        background: $foreground 10%;
+        color: $text;
+    }
+
+    CollapsibleTitle:focus {
+        background: $accent;
     }
     """
 
-    class Title(Horizontal):
-        DEFAULT_CSS = """
-        Title {
-            width: 100%;
-            height: auto;
-        }
+    BINDINGS = [Binding("enter", "toggle", "Toggle collapsible", show=False)]
 
-        Title:hover {
-            background: grey;
-        }
+    collapsed = reactive(False)
```

Comment:
I find it a bit odd that `Collapsible.collapsed` starts off as `True` and this one starts off as `False`!

Reply:
Is a bit. Changed to match the default.